src/theory/quantifiers/sygus/sygus_interpol.cpp

```diff
@@ -0,0 +1,95 @@
+/*********************                                                        */
+/*! \file sygus_interpol.cpp
+ ** \verbatim
+ ** Top contributors (to current version):
+ **   Ying Sheng
+ ** This file is part of the CVC4 project.
+ ** Copyright (c) 2009-2020 by the authors listed in the file AUTHORS
+ ** in the top-level source directory) and their institutional affiliations.
+ ** All rights reserved.  See the file COPYING in the top-level source
+ ** directory for licensing information.\endverbatim
+ **
+ ** \brief Implementation of sygus interpolation utility, which
+ ** transforms an arbitrary input into an interpolation problem.
+ **/
+
+#include "theory/quantifiers/sygus/sygus_interpol.h"
+
+#include "expr/datatype.h"
+#include "expr/dtype.h"
+#include "expr/node_algorithm.h"
+#include "expr/sygus_datatype.h"
+#include "options/smt_options.h"
+#include "printer/sygus_print_callback.h"
+#include "theory/datatypes/sygus_datatype_utils.h"
+#include "theory/quantifiers/quantifiers_attributes.h"
+#include "theory/quantifiers/quantifiers_rewriter.h"
+#include "theory/quantifiers/sygus/sygus_grammar_cons.h"
+#include "theory/quantifiers/term_util.h"
+#include "theory/rewriter.h"
+
+using namespace CVC4::kind;
+using namespace std;
```

Comment:
We've been leaning more towards explicitly using the prefixes `std` and `kind` in the code, rather than assuming the namespace.

Reply:
Yes, we strongly encourage to not `using namespace std`. We should probably make this a style guideline and actually not permit it, what do you think @4tXJ7f @mpreiner @ajreynol ?

Reply:
I agree with you. This is been added accidentally.